kernel/packages/shared/meta/index.ts

```diff
@@ -20,3 +23,20 @@ export async function ensureMetaConfigurationInitialized(): Promise<void> {
     })
   })
 }
+
+export async function waitForMessageOfTheDay(): Promise<MessageOfTheDayConfig | null> {
+  const store: Store<RootState> = globalThis.globalStore
+  const worldConfig = store.getState().meta.config.world
```

Comment:
The best practice would be to implement a selector for this, if there is none already

Reply:
yes, you're right I'll change it